.teamcity/settings.kts

```diff
@@ -79,6 +79,7 @@ object Build_Release: BuildType({
         param("env.RG_DEV", "ESdCDPSBDMK8SDev")
         param("env.TARGET", "main")
         param("env.BRANCH", "main")
+        param("env.NEXT_CONTENT_GRAPHQL", "%vault:dts-secrets-dev/digitalCentre!/NEXT_CONTENT_GRAPHQL%")
```

Comment:
`param("env.NEXT_CONTENT_GRAPHQL", "%vault:dts-secrets-dev/scrumPoker!/NEXT_CONTENT_GRAPHQL%")`

Just need to change the digitalCentre to scrumPoker

Same for the other two places